javascript/core/functions-iii/context-in-javascript.md

```diff
@@ -19,6 +19,11 @@ standards:
 
   javascript.execution-context.3: 10
 
+aspects:
```

Comment:
This shouldn't be `obscura`, consider `introduction` as well. You will definitely see this as a beginner, in ES6 classes or in code snippets.